charts/kubeflow-trainer/templates/controller/deployment.yaml

```diff
@@ -0,0 +1,113 @@
+{{/*
+Copyright 2024 The Kubeflow authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "trainer.controller.deployment.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "trainer.controller.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.controller.replicas }}
+  selector:
+    matchLabels:
+      {{- include "trainer.controller.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "trainer.controller.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+      - name: controller
+        image: {{ include "trainer.image" . }}
+        {{- with .Values.image.pullPolicy }}
+        imagePullPolicy: {{ . }}
+        {{- end }}
+        command:
+        - /manager
+        args:
+        - --metrics-bind-address=0
+        - --metrics-secure=true
+        - --health-probe-bind-address=:8081
+        - --enable-http2=false
+        - --webhook-server-port=9443
+        - --webhook-service-name={{ include "trainer.controller.service.name" . }}
+        - --webhook-secret-name={{ include "trainer.webhook.secret.name" . }}
+        - --leader-elect=false
```

Comment:
This is needed to be updated according to this:
https://github.com/kubeflow/trainer/blob/master/manifests/base/manager/manager.yaml#L5